types/webextension-polyfill/namespaces/sessions.d.ts

```diff
@@ -98,15 +93,15 @@ export namespace Sessions {
          * @param key The key which corresponds to the value being set.
          * @param value The value being set.
          */
-        setTabValue(tabId: number, key: string, value: any): Promise<void>;
+        setTabValue(tabId: number, key: string, value: unknown): Promise<void>;
 
         /**
          * Retrieve a value that was set for a given key on a given tab.
          *
          * @param tabId The id of the tab whose value is being retrieved from.
          * @param key The key which corresponds to the value.
          */
-        getTabValue(tabId: number, key: string): Promise<any>;
+        getTabValue(tabId: number, key: string): Promise<unknown>;
```

Comment:
This seems like another change from `any` to `unknown` in an output position.

Reply:
Same reason: only the caller knows what type it's supposed to be.